examples/FeatureFlagDemo/Controllers/HomeController.cs

```diff
@@ -15,10 +15,13 @@ namespace FeatureFlagDemo.Controllers
     public class HomeController : Controller
     {
         private readonly IFeatureManager _featureManager;
+        private readonly IFeatureVariantManager _variantManager;
 
-        public HomeController(IFeatureManagerSnapshot featureSnapshot)
+        public HomeController(IFeatureManagerSnapshot featureSnapshot,
+                              IFeatureVariantManager variantManager)
         {
             _featureManager = featureSnapshot;
+            _variantManager = variantManager;
```

Comment:
I don't see `_variantManager` is used anywhere.

Reply:
Removed.